help_to_heat/middleware.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+from django.core.handlers.wsgi import WSGIRequest
+
+middleware_logger = logging.getLogger(__name__)
+requests_logger = logging.getLogger("requests")
```

Comment:
Interested in why we're using two different loggers in general, and why we've defined a custom one for requests?

Reply:
the custom logger in general allows for things such as specifying the formatter which could come in handy. will probably migrate to using a single logger throughout the app (or overriding an existing one & using that in all places)